docs/integrations/context-and-outputs.md

```diff
@@ -1076,6 +1076,93 @@ outputs:
 }
 ```
 
+---
```

Comment:
It seems that there's an extra newline which can be seen in the preview of the page, please remove it.